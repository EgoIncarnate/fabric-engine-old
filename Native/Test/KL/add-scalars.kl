operator entry()
{
  report 3.4 + 8.7;
}
