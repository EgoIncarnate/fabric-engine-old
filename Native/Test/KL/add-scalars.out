GlobalList
  Function( 'entry' ('entry'), void )
    ParamList
    CompoundStatement
      StatementList
        Report
          BinOp( ADD )
            ConstScalar( 3.4 )
            ConstScalar( 8.7 )
12.1
